sdk/maps/azure-maps-search/samples/async_samples/sample_get_polygon_async.py

```diff
@@ -7,31 +7,33 @@
 # --------------------------------------------------------------------------
 
 """
-FILE: sample_get_point_of_interest_categories_async.py
+FILE: get_polygon_async.py
 DESCRIPTION:
-    This sample demonstrates get POI categories.
+    This sample demonstrates how to search polygons.
 USAGE:
-    python sample_get_point_of_interest_categories_async.py
+    python get_polygon_async.py
 
     Set the environment variables with your own values before running the sample:
     - AZURE_SUBSCRIPTION_KEY - your subscription key
 """
 import asyncio
 import os
 
-subscription_key = os.getenv("AZURE_SUBSCRIPTION_KEY")
+subscription_key = os.getenv("AZURE_SUBSCRIPTION_KEY", "your subscription key")
 
-async def get_point_of_interest_categories_async():
+async def get_polygon_async():
     from azure.core.credentials import AzureKeyCredential
     from azure.maps.search.aio import MapsSearchClient
 
     maps_search_client = MapsSearchClient(credential=AzureKeyCredential(subscription_key))
-
     async with maps_search_client:
-        result = await maps_search_client.get_point_of_interest_categories()
+        result = await maps_search_client.get_polygon(
+            coordinates=[-122.204141, 47.61256],
+            result_type="locality",
```

Comment:
Can we use `BoundaryResultTypeEnum.xxx` to some or all the samples? (same for `reverseGeocodeResultTypeEnum`)